### ISSUE TITLE
Any way to Select dot on timeline

### ISSUE THREAD

**Issue by dobromir-hristov** [almende/vis#1514]
Hello. Is there a way to select the dots on the time line instead of the text above it? 
I couldnt find any relation between the dots and the items in the DOM, maybe i`m missing something?

**Comment by josdejong** [comment]
You're right, you can't click the dots to select an item, an item is only selected when you click the item contents.

**Comment by dobromir-hristov** [comment]
So there is no way to select an item by clicking on the timeline dot. I searched how the dots were created or how they are selected upon selection of an item so I could reverse engineer it, yet I got a dead end there. The only method I figured out was getting the dot`s index and based on that find the item, but I guess that is not a good method?

**Comment by josdejong** [comment]
> So there is no way to select an item by clicking on the timeline dot

That's correct. We could implement it though ;)

The Timeline uses event delegation to select items, see methods [ItemSet.prototype._onSelectItem](https://github.com/almende/vis/blob/master/lib/timeline/component/ItemSet.js#L1653-L1679) and [ItemSet.prototype.itemFromTarget](https://github.com/almende/vis/blob/master/lib/timeline/component/ItemSet.js#L1861).

**Comment by dobromir-hristov** [comment]
Well if you do, you will make my life a whole lot easier ... Will wait for a reply.

**Comment by josdejong** [comment]
We will not have time for this any time soon. Feel free to contribute if you like.

**Comment by yotamberk** [comment]
Solved in v4.17.0

**Comment by mojoaxel** [comment]
Selecting an item by clicking on the dot does not work for me. At least not on mobile.

**Comment by yotamberk** [comment]
Whoops... I thought the dot on the point item... This sounds more like a feature request
